lib/src/view/analysis/analysis_screen.dart

```diff
@@ -377,6 +377,63 @@ class _Board extends ConsumerStatefulWidget {
 class _BoardState extends ConsumerState<_Board> {
   ISet<cg.Shape> userShapes = ISet();
 
+  ISet<cg.Shape> _computeBestMoveShapes(IList<MoveWithWinningChanges> moves) {
+    // Scale down all moves with index > 0 based on how much worse their winning chances are compared to the best move
+    // (assume moves are ordered by their winning chances, so index==0 is the best move)
+    double scaleArrowAgainstBestMove(int index) {
+      final bestMove = moves[0];
+      final winningDiffComparedToBestMove =
+          bestMove.winningChances - moves[index].winningChances;
+      const double minScale = 0.25;
+      // Force minimum scale if...
+      // 1) The best move is significantly better than this move
+      // 2) The signs differ, e.g. the best move is winning while this move draws (or even looses)
+      if (winningDiffComparedToBestMove > 0.3 ||
+          bestMove.winningChances.sign != moves[index].winningChances.sign) {
+        return minScale;
+      }
+      return math.max(
+        minScale,
+        1.4 - 5.0 * winningDiffComparedToBestMove,
```

Comment:
I recently pushed a change to the `flutter-chessground` package: the `scale` is now constrained like: `0 < scale <= 1.0`.

So we want to clamp the value to these limits.

Also the 1.4 and 5 numbers come a bit out of nowhere; a comment or having names for them would be welcome.

Reply:
Good point, these values were determined by the "play around and see what looks good" method, I'll give them better names (and also adapt to the changed scale limits)